src/elements/a-gltf-entity.js

```diff
@@ -229,6 +232,7 @@ AFRAME.registerElement("a-gltf-entity", {
           // If the src attribute is a selector, get the url from the asset item.
           if (src && src.charAt(0) === "#") {
             const assetEl = document.getElementById(src.substring(1));
+            if (!assetEl) { return; }
```

Comment:
This should at least log an error. It might also make sense to have it emit `model-error` but not sure about that one.